ci/run-benchmark.yaml

```diff
@@ -0,0 +1,59 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: theorf/google-cloud-sdk-helm
+inputs:
+  - name: eq-survey-runner-benchmark
+params:
+  PROJECT_ID:
+  REGION:
+  REQUESTS_JSON:
+  LOCUST_OPTIONS:
+  RUNNER_FULLY_QUALIFIED_DOMAIN_NAME:
+  DOCKER_REGISTRY:
+  IMAGE_TAG:
+  USER_WAIT_TIME_MIN_SECONDS:
+  USER_WAIT_TIME_MAX_SECONDS:
+  PARALLELISM:
+  TIMEOUT:
+run:
+  path: bash
+  args:
+    - -exc
+    - |
+      apt-get install -y procps
+
+      cat >~/gcloud-service-key.json <<EOL
+      ((gcp.service_account_json))
+      EOL
+
+      gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
+
+      cd eq-survey-runner-benchmark
+
+      helm init --client-only
+      helm plugin install https://github.com/rimusz/helm-tiller
+
+      GCS_OUTPUT_BUCKET=${PROJECT_ID}-benchmark-outputs
+      RUNTIME_DATE_STRING="$(date +'%Y-%m-%dT%H:%M:%S')"
+
+      gcloud container clusters get-credentials runner-benchmark --region ${REGION} --project ${PROJECT_ID}
+
+      helm tiller run \
+        helm upgrade --install \
+        runner-benchmark \
+        k8s/helm \
+        --set requestsJson="${REQUESTS_JSON}" \
+        --set locustOptions="${LOCUST_OPTIONS}" \
+        --set host=https://${RUNNER_FULLY_QUALIFIED_DOMAIN_NAME} \
+        --set container.image=${DOCKER_REGISTRY}/eq-survey-runner-benchmark:${IMAGE_TAG} \
+        --set userWaitTimeMinSeconds=${USER_WAIT_TIME_MIN_SECONDS} \
+        --set userWaitTimemaxSeconds=${USER_WAIT_TIME_MAX_SECONDS} \
+        --set parallelism=${PARALLELISM} \
+        --set output.bucket=${GCS_OUTPUT_BUCKET} \
+        --set output.directory="timed-schedule/${RUNTIME_DATE_STRING}"
```

Comment:
We should pass in the directory from concourse so it will be:
```yaml
--set output.directory="daily-test/${RUNTIME_DATE_STRING}"
```
or
```yaml
--set output.directory="stress-test/${RUNTIME_DATE_STRING}"
```